scipy/signal/_upfirdn.py

```diff
@@ -92,7 +92,7 @@ def apply_filter(self, x, axis=-1, mode='constant', cval=0):
         output_len = _output_len(self._h_len_orig, x.shape[axis],
                                  self._up, self._down)
         # Explicit use of np.int64 for output_shape dtype avoids OverflowError
-        # when allocating large array on platforms where np.int_ is 32 bits
+        # when allocating large array on platforms where long is 32 bits.
```

Comment:
The argument here should use `intp` then really.

Reply:
Right - I addressed it in https://github.com/scipy/scipy/pull/19394.